spec/cell_spec.rb

```diff
@@ -0,0 +1,41 @@
+require './lib/ship'
+require './lib/cell'
+
+RSpec.describe Cell do
+  before(:each)do
+    @cruiser = Ship.new("Cruiser", 3)
+    @cell = Cell.new("B4")
+  end
+
+  describe '#initialize' do
+    it 'exists' do
+      expect(@cell).to be_a(Cell)
+    end
+
+    it 'has a coordinate' do
+      expect(@cell.coordinate).to eq("B4")
+    end
+  end
+  
+  describe '#ship' do
+    it 'has defaults to nil' do
```

Comment:
little typo in the title here I can fix in my next push!

Reply:
That sounds great and good catch, thanks!